heartbeat/azure-lb

```diff
@@ -15,11 +15,16 @@
 # Initialization:
 : ${OCF_FUNCTIONS_DIR=${OCF_ROOT}/lib/heartbeat}
 . ${OCF_FUNCTIONS_DIR}/ocf-shellfuncs
-
+. ${OCF_FUNCTIONS_DIR}/ocf-distro
 # Defaults
 OCF_RESKEY_nc_default="/usr/bin/nc"
 OCF_RESKEY_port_default="61000"
 
+# On SLES we use socat
+if [ $( get_release_id ) = "SUSE" ]; then
```

Comment:
@varkoly Some comments here. The usage of `socat` is not SUSE specific, it can be used in any distro, so this check does not make sense here.